src/components/ClearingsUploader/ClearingsUploader.js

```diff
@@ -109,7 +133,9 @@ export const ClearingsUploader = ({
     <ClearingsUploaderWrapper>
       <FileUploadWrapper>
         <StyledP>{messages.step_1}</StyledP>
-        <Select id="provider">{ListItems}</Select>
+        <Select id="provider" onChange={onSelect}>
```

Comment:
You need to pass `value={provider}` and receive the provider as a prop to make this useful.

Reply:
done